topics/epigenetics/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -322,7 +323,7 @@ Since in this tutorial we are interested in assessing H3K4me3, H3K27me3 and CTCF
 >
 >    Using these parameters, the tool will take bins of 1000 bp separated by 500 bp on the chromosome X. For each bin the overlapping reads in each sample will be computed and stored into a matrix.
 >
-> 4. **plotCorrelation** {% icon tool %} with the following parameters
+> 4. {% tool [plotCorrelation](toolshed.g2.bx.psu.edu/repos/bgruening/deeptools_plot_correlation/deeptools_plot_correlation/3.5.1.0.0) %} with the following parameters:
```

Comment:
Similar comment as previously made for the `multiBamSummary` tool, can we use the latest version?